qiwis.py

```diff
@@ -609,6 +620,23 @@ def update_result(self, request: str, result: QiwiscallResult, discard: bool = T
         logger.debug("Qiwiscall result is updated: %s", _result)
 
 
+def set_global_constant_namespace(constants: Mapping[str, JsonType]) -> Tuple:
+    """Creates a immutable namedtuple and sets it as the global constant namespace.
+
+    Args:
+        constants: A mapping source for the global constant namespace.
+          The key-values become the constant name-values and hence the keys
+          should be a valid namedtuple field name.
+
+    Returns:
+        The created namedtuple, which is the global constant namespace.
+    """
+    constantnamespace = namedtuple("constantnamespace", constants.keys())

```

Comment:
How about starting the variable name with a capital letter?
In the [reference](https://docs.python.org/3/library/collections.html#collections.namedtuple), it looks common.

Reply:
Oh, it makes sense! However, then we should change the pylint variable naming convention anyway.
Well, defining a class like that is quite okay in Python, so it might not be a problem.
If you agree, I will update the pylintrc files.

Reply:
Good! Please update the pylintrc files!